src/App.tsx

```diff
@@ -1,27 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+import 'bulma/css/bulma.css';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  // const [currentItems, setCurrentItems] = useState(items);
+  const [perPage, setPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const lastItemIndex = perPage * currentPage;
+  const firstItemIndex = lastItemIndex - perPage;
```

Comment:
The naming of `lastItemIndex` and `firstItemIndex` might be misleading since they represent indices and not actual items. Consider renaming them to `lastItemIndexExclusive` and `firstItemIndexInclusive` to clarify that the `slice` method includes the start index and excludes the end index.